Feat/352 updated sidebar

dexturr opened the pull request:
Closes #352 
Closes #808

Needs #785 first.

![Screenshot 2021-11-11 at 11 32 57](https://user-images.githubusercontent.com/26136207/141291389-85b80a2c-308a-44f1-a6db-1c74377c7a63.png)

campbellssource commented:
issue (blocking): 
The Link to connect you eth wallet is white text. it should be black.
in this screenshot I am hovering over the white text
<img width="486" alt="Screenshot 2021-11-15 at 13 40 16" src="https://user-images.githubusercontent.com/13255539/141791603-ea70b25b-055c-40ea-9a86-4fe82cce9d90.png">
here without hover
<img width="493" alt="Screenshot 2021-11-15 at 13 40 11" src="https://user-images.githubusercontent.com/13255539/141791936-c20645d7-91b8-4efd-a8b4-0fe1b51f90ab.png">

campbellssource commented:
Issue (blocking):
The `__actions` div should be 100% width. or is should have the effect that each button shares approx 50% of the container.
<img width="428" alt="Screenshot 2021-11-15 at 13 43 25" src="https://user-images.githubusercontent.com/13255539/141792342-20fe81e8-a7eb-4046-936c-a83cb7b6ad7f.png">

campbellssource commented:
Request (not blocking): The VEGA in Vesting contract should not appear if sum value is 0. i.e. you do not have any tokens vesting.
<img width="523" alt="Screenshot 2021-11-15 at 13 54 36" src="https://user-images.githubusercontent.com/13255539/141793905-38dee3c8-ada9-4e60-99dd-218e5ebaf8a3.png">

campbellssource commented:
Request (non-blocking): link hover text is same as background on hover for the list of Vega keys.
hit "Change key" a list of keys appears (it should appear beneath the "change key" link but that isn't my main beef).
a list of keys appears. but the hover state is black
<img width="423" alt="Screenshot 2021-11-15 at 14 09 07" src="https://user-images.githubusercontent.com/13255539/141796164-f4dc771e-74e8-4393-b382-cdb9e577ffcd.png">

campbellssource commented:
question (non-blocking): @dexturr does the list of assets just show the general balance at the moment, or does it sum up all accounts in that asset?

<img width="619" alt="Screenshot 2021-11-15 at 14 18 58" src="https://user-images.githubusercontent.com/13255539/141797341-f7cd17f7-2af8-4781-a6d0-7997357e5848.png">

mattrussell36 commented on src/components/wallet-card/wallet-card.tsx:
```diff
@@ -62,3 +92,55 @@ export const WalletCardActions = ({
+  border,
+  dark,
+}: WalletCardAssetProps) => {
+  const [integers, decimalsPlaces] = React.useMemo(() => {
```
Make this a custom hook (or component) to avoid repeating this code?

mattrussell36 reviewed (changes requested):
All good. Just one thing I think worth changing is to avoid repeating the integer/decimal splitting code

dexturr commented:
@mattrussell36 is the intrest of throwing this to QA as fast as possible and I have addressed your comments + @campbellssource's I'm going to merge this assuming your approval 👍
